.github/workflows/update-docs-webhook.yaml

```diff
@@ -0,0 +1,21 @@
+name: Update docs webhook
+on:
+  push:
+    branches:
+      - master
+      - branch/v*
+  workflow_dispatch:
+
+jobs:
+  update-webhook:
+    name: Update docs webhook
+    runs-on: ubuntu-latest
+    environment: update-docs
+    steps:
+      - name: Call deployment webhook
+        env:
+          WEBHOOK_URL: ${{ secrets.DOCS_DEPLOY_HOOK }}
+        run: |
+          if curl --silent --fail --show-error "$WEBHOOK_URL" > /dev/null; then
```

Comment:
This is going to silently swallow a curl error if it cannot hit the docs endpoint. Is this intended? If so, perhaps a comment as to why?